core/designsystem/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,6 @@
     <string name="OTFPasswordLabel">Password</string>
     <string name="OTFPasswordContentDescriptionError">Error</string>
     <string name="OTFPasswordMessageError">At least 6 characters</string>
+    <string name="contact_phone">Contact phone</string>
```

Comment:
I recommend you use these two strings `contact_phone` and `mask`, because is not used in the code